src/tests/ftest/util/telemetry_test_base.py

```diff
@@ -84,3 +84,181 @@ def verify_telemetry_list(self, with_pools=False):
             self.fail("\n".join(errors))
 
         self.log.info("Test PASSED")
+
+    def get_min_max_mean_stddev(self, suffix, num_targets):
+        """Get four lists of min, max, mean, stddev.
+
+        Sample get_pool_metrics output.
+        {
+            'engine_io_ops_dkey_enum_latency_min': {
+                'wolf-31': {
+                    '0': {
+                        '0': 6,
+                        '1': 13,
+                        '2': 7,
+                        '3': 6,
+                        '4': 6,
+                        '5': 8,
+                        '6': 10,
+                        '7': 12
+                    }
+                },
+                'wolf-32': {
+                    '1': {
+                        '0': 10,
+                        '1': 9,
+                        '2': 7,
+                        '3': 10,
+                        '4': 18,
+                        '5': 9,
+                        '6': 6,
+                        '7': 11
+                    }
+                }
+            },
+            ...
+
+        Args:
+            suffix (str): Metrics suffix for the metric that has min, max,
+                mean, or stddev at the end.
+            num_targets (int): Number of targets.
+
+        Returns:
+            dict: min, max, mean, stddev from each target in a dict.
+                {
+                    "min": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "max": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "mean": [0, 1, 2, 3, 4, 5, 6, 7...],
+                    "stddev": [0, 1, 2, 3, 4, 5, 6, 7...]
+                }
+                Note that all the stats should have the same rank order for the
+                verification to work. Not using the sample input values.
```

Comment:
What do you mean by "Not using the sample input values"?

Reply:
It means not using the values from the sample input towards the beginning of the method description in get_min_max_mean_stddev(). The input values are 6, 13, 7, 6, 6, etc., but I'm using 0, 1, 2, 3... in the output. I removed this comment since it looks like it's confusing.